src/Sarif.UnitTests/Baseline/ResultMatching/ResultMatchingBaselinerTests.cs

```diff
@@ -58,11 +58,18 @@ public void ResultMatchingBaseliner_BaselinesTwoSimpleSarifLogs()
                 AbsentResultProperties.Should().ContainKey("Run");
                 AbsentResultProperties["Run"].Should().BeEquivalentTo(baselineLog.Runs[0].InstanceGuid);
 
-                calculatedNextBaseline.Runs[0].Results.Where(r => r.BaselineState == BaselineState.Existing).Should().HaveCount(currentLog.Runs[0].Results.Count - 1);
 
-                calculatedNextBaseline.Runs[0].Results.Where(r => r.BaselineState == BaselineState.Existing).First().TryGetProperty(SarifLogResultMatcher.ResultMatchingResultPropertyName, out Dictionary<string, string> CurrentResultProperties).Should().BeTrue();
-                CurrentResultProperties.Should().ContainKey("Run");
-                CurrentResultProperties["Run"].Should().BeEquivalentTo(currentLog.Runs[0].InstanceGuid);
+                int existingCount = currentLog.Runs[0].Results.Count - 1;
+                calculatedNextBaseline.Runs[0].Results.Where(r => r.BaselineState == BaselineState.Existing).Count().Should().Be(existingCount);
+
+                if (existingCount > 0)
```

Comment:
>existingCount [](start = 20, length = 13)

This is a fix for a bug in a test that produces random content (that happened to fail intermittently in one of my validation runs)